linode/manager
# M3-5353: Volume Create - remove checkout sidebar

## Description

This PR removes the sidebar from the Volume Create page, moves the create button to below the form, and moves the pricing below the size textfield. The new storage options banner inside the Volume Create form is also changed to green.

Also fixed a minor bug in `Breadcrumb.tsx` where classes aren't actually being applied.

## How to test

- Make sure you can create a Volume via `/volumes/create` and from a Linode
- Check for any regressions in resizing a Volume
